packages/foodie-backend/src/utils/encryption.js

```diff
@@ -0,0 +1,33 @@
+import crypto from 'crypto'
```

Comment:
what are we going to be needing encryption for ?

Reply:
For encrypting some important strings text data